lib/PuppeteerSharp/TargetInfo.cs

```diff
@@ -7,54 +9,51 @@ namespace PuppeteerSharp
     /// </summary>
     public class TargetInfo
     {
-        internal TargetInfo()
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TargetInfo"/> class.
+        /// </summary>
+        public TargetInfo()
```

Comment:
Remove to use implicit default constructor.

Reply:
done!